src/components/cesium_map/api/spatial.js

```diff
@@ -322,6 +323,14 @@ export class ISamplesSpatial {
     // 10 the minimum height for the points so the users wouldn't zoom to the ground.
     this.viewer.scene.screenSpaceCameraController.maximumZoomDistance = 20000000;
     this.viewer.scene.screenSpaceCameraController.minimumZoomDistance = 10;
+    // set camera inital position
+    this.viewer.camera.setView({
+      destination: Cesium.Cartesian3.fromDegrees(-149.8169236266867, -17.451466233002286, 2004.7347996772614),
+      orientation: {
+        heading: Cesium.Math.toRadians(201.84408760864753),
+        pitch: Cesium.Math.toRadians(-20.853642866175978),
+      }
+    });
```

Comment:
Set the initial camera view to avoid rerendering the initial points.

Reply:
You can use an instance of SpatialView for this, like:
```
this.viewer.camera.setView({destination:view.destination(), orientation:view.orientation()})
```
or even better, add a `getView()` method to `SpatialView` to return destination and orientation in a single call.

Then pass in an instance of `SpatialView` to the `ISamplesSpatial` constructor as an `initialView` parameter. This could be one of the already defined SpatialViews in `cesium_UI.js`